lib/rules/web/admin_console/4.8/dashboards.xyaml

```diff
@@ -667,6 +667,20 @@ check_charts_in_project_utilization:
   - selector:
       xpath: //h4[text()='Pod count']/ancestor::div[@data-test-id='utilization-item']//div[contains(@class,'pf-c-chart')]
     timeout: 40
+
+check_cpu_chart_orange_line:
+  params:
+      graph_title: CPU
+  action: check_cluster_utilization_chart_with_orange_line
+check_memory_chart_orange_line:
+  params:
+      graph_title: Memory
+  action: check_cluster_utilization_chart_with_orange_line
+check_cluster_utilization_chart_with_orange_line:
+  elements:
+  - selector:
+      xpath: //h4[text()='<graph_title>']/ancestor::div[@data-test-id='utilization-item']//div[contains(@class,'pf-c-chart')]/*[name()='svg']/*[name()='g'][2]/*[name()='g'][2]/*[name()='g']/*[name()='path' and contains(@style, 'rgb(236, 122, 8)')]
```

Comment:
The xpath expression here is unstable(easily fail) since you use too many relative paths

Reply:
Have you ever tried `hover` in  https://github.com/openshift/verification-tests/blob/master/doc/webauto.adoc#op

Reply:
Let me try the hover action

Reply:
Done for update